functions/jira/auth/token.ts

```diff
@@ -0,0 +1,48 @@
+import "dotenv/config";
+import { callAPI } from "../callAPI";
+import { API_OPS } from "../APIOperations";
+import { HEADERS } from "../header";
+import { Request, Response } from "express";
+import { OAuth2Client } from "@badgateway/oauth2-client";
+import { allowCors } from "../../_utils/helpers";
+
+// TODO: frontend should probably get these consts from here
+const REDIRECT_URI = "http://localhost:5173/callback";
+const SCOPE = ["read:jira-work", "read:jira-user"];
+const BACKEND_URL = "https://local.functions.nhost.run/v1/jira/auth/token";
```

Comment:
These URLs are most likely going to change once the website gets deployed to Netlify, how do you plan on solving that change?